<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-2RIICMC" failures="3" tests="52" name="Default test" time="2.266" errors="0" timestamp="2024-08-28T12:52:45 EET">
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.046"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.01"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.016"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.038"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.009"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.011"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.009"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.011"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.013"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.014"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.011"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.009"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.004"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.01"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.733">
    <failure type="java.sql.SQLIntegrityConstraintViolationException" message="Duplicate entry &amp;apos;499&amp;apos; for key &amp;apos;customers.PRIMARY&amp;apos;">
      <![CDATA[java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '499' for key 'customers.PRIMARY'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:109)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
at com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1537)
at com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2384)
at com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1410)
at DBTesting.MyTestCases.AdddataToThedataBase(MyTestCases.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- AdddataToThedataBase -->
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.003">
    <failure type="java.sql.SQLIntegrityConstraintViolationException" message="Duplicate entry &amp;apos;301&amp;apos; for key &amp;apos;customers.PRIMARY&amp;apos;">
      <![CDATA[java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '301' for key 'customers.PRIMARY'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:109)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
at com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1537)
at com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2384)
at com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1410)
at DBTesting.MyTestCases.AdddataToThedataBase(MyTestCases.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- AdddataToThedataBase -->
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.008">
    <failure type="java.sql.SQLIntegrityConstraintViolationException" message="Duplicate entry &amp;apos;64&amp;apos; for key &amp;apos;customers.PRIMARY&amp;apos;">
      <![CDATA[java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '64' for key 'customers.PRIMARY'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:109)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
at com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1537)
at com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2384)
at com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1410)
at DBTesting.MyTestCases.AdddataToThedataBase(MyTestCases.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- AdddataToThedataBase -->
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.006"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.004"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.007"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.015"/>
  <testcase classname="DBTesting.MyTestCases" name="AdddataToThedataBase" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="updateData" time="0.005"/>
  <testcase classname="DBTesting.MyTestCases" name="myFirstTestToGetData" time="0.082"/>
  <testcase classname="DBTesting.MyTestCases" name="DeleteData" time="0.0">
    <ignored/>
  </testcase> <!-- DeleteData -->
</testsuite> <!-- Default test -->
